tests/integration/tests.mk

```diff
@@ -42,6 +42,7 @@ ifeq ($(IP_FAMILY),ipv6)
 	support_ipv6 = yes
 else ifeq ($(IP_FAMILY),dual)
 	support_ipv6 = yes
+	_INTEGRATION_TEST_FLAGS += --istio.test.enableDualStack
 endif
 ifdef support_ipv6
 	_INTEGRATION_TEST_SELECT_FLAGS:="$(_INTEGRATION_TEST_SELECT_FLAGS),-ipv4"
```

Comment:
Do we want to remove the ipv4 in the case of dual stack?

Reply:
Hi @ericvn, thanks for your comment. I think `dual stack` should support `ipv4` to make sure that the original integration test cases can be available as well. Does it make sense?

Reply:
Correct. I think you current line 48 removes ipv4 in the cause of IP_FAMILY being dual. Just making sure that's desired.

Reply:
okay, I got it 
![image](https://user-images.githubusercontent.com/4101246/190370615-216f48ef-9c9a-4f6f-a391-e1a73e6ac8b3.png)
The ipv6 request can not be handled because the dual stack feature still can not be supported in master branch.